src/autogluon/bench/cloud/aws/batch_stack/stack.py

```diff
@@ -231,3 +230,9 @@ def __init__(self, scope: Construct, id: str, static_stack: StaticResourceStack,
         )
         core.CfnOutput(self, "JobQueueARN", value=job_queue.job_queue_arn)
         core.CfnOutput(self, "JobDefinitionARN", value=job_definition.job_definition_arn)
+        core.CfnOutput(
+            self,
+            "EcrRepositoryName",
+            value=docker_image_asset.repository.repository_name,
+        )
+        core.CfnOutput(self, "ImageUri", value=docker_image_asset.image_uri)
```

Comment:
Why did these have to be added? I guess I'm surprised it worked before without them.

Reply:
This is to expose the resource info after deploying the stack so that we can later query it when describe the stack. We didn't need to remove image from ECR before.